pylearn2/datasets/hdf5.py

```diff
@@ -2,338 +2,467 @@
 Objects for datasets serialized in HDF5 format (.h5).
 """
 
-__author__ = "Steven Kearnes"
-__copyright__ = "Copyright 2014, Stanford University"
+__author__ = "Francesco Visin"
+__copyright__ = "Copyright 2015, Politecnico di Milano"
```

Comment:
Why do we add copyright anyway? Don't all the files fall under the same BSD-3 license, which lists as author "Université de Montréal"? Copyright would suggest that you have some sort of exclusive rights over the code you wrote, but by submitting it to Pylearn2 you waive all of those rights to UdeM as part of the license I think.